libnavigation-base/src/test/java/com/mapbox/navigation/base/internal/utils/DirectionsRouteMissingConditionsCheckTest.kt

```diff
@@ -0,0 +1,106 @@
+package com.mapbox.navigation.base.internal.utils
+
+import com.mapbox.api.directions.v5.models.DirectionsRoute
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.mockkObject
+import io.mockk.unmockkObject
+import junit.framework.Assert.assertTrue
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import java.net.URL
+
+class DirectionsRouteMissingConditionsCheckTest {
+
+    private val exclusiveQueries = mapOf(
+        "engine" to "electric",
+        "key1" to "value1",
+    )
+
+    @Before
+    fun setup() {
+        mockkObject(QueriesProvider)
+        every { QueriesProvider.provideQueries() } returns exclusiveQueries
+    }
```

Comment:
If `exclusiveQueries` was an argument to `DirectionsRouteMissingConditionsCheck.checkDirectionsRoute` it wouldn't require a static mocking to be tested

Reply:
DirectionsRouteMissingConditionsCheck is stateless and it's invoked from static functions, let's keep it as the object